set.mm

```diff
@@ -27969,6 +27998,16 @@ choice between (what we call) a "definitional form" where the shorter
       ( wral ralbii ) ADFHBDFHCEABDFGII $.
   $}
 
+  ${
+    $d x ph $.
+    ralbidv2.1 $e |- ( ph -> ( ( x e. A -> ps ) <-> ( x e. B -> ch ) ) ) $.
+    $( Formula-building rule for restricted universal quantifier (deduction
+       form).  (Contributed by NM, 6-Apr-1997.) $)
+    ralbidv2 $p |- ( ph -> ( A. x e. A ps <-> A. x e. B ch ) ) $=
+      ( cv wcel wi wal wral albidv df-ral 3bitr4g ) ADHZEIBJZDKPFICJZDKBDELCDFL
+      AQRDGMBDENCDFNO $.
+  $}
+
```

Comment:
It looks like you still have not moved back `ralbidv2` to its original location.